src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import duke.exception.EmptyDescription;
+import duke.exception.InvalidTypeOfTask;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import java.io.IOException;
+import java.util.Scanner;
+
```

Comment:
Careful about the import statements ordering. Similar issues in other files, for example TaskList.java